restapi/pagination.mdx

```diff
@@ -21,23 +21,19 @@ Axiom uses cursor-based pagination for these endpoints. The parameters and mecha
 For the Run Query endpoint, add the `includeCursor` parameter to the body of your API request and set its value to true.
```

Comment:
You don't need the `includeCursor` at all

Reply:
Indeed! Could you please check this commit? Fixed other things as well. https://github.com/axiomhq/docs/pull/61/commits/393deeb9ab8dbd6b5e7040f8ec8495951b8df4e4